Mathlib/RingTheory/LaurentSeries.lean

```diff
@@ -38,6 +41,47 @@ variable {R : Type*}
 
 namespace LaurentSeries
 
+section HasseDeriv
+
+variable {V : Type*} [AddCommGroup V] [Semiring R] [Module R V]
+
+theorem hasseDeriv_bdd_below (k : ℕ) (f : LaurentSeries V) (m : ℤ) (h : m < f.order - k) :
+    (fun (n : ℤ) ↦ Ring.choose (n + k) k • HahnSeries.coeff f (n + k)) m = 0 := by
+  simp only
+  rw [@lt_sub_iff_add_lt] at h
+  rw [coeff_eq_zero_of_lt_order h, smul_zero]
```

Comment:
I think this lemma is too close to `coeff_eq_zero_of_lt_order` to justify its existence. I see you are using it for the definition below, but it can be inlined.
```suggestion
```